rfcs/0000-top-level-assumptions.md

```diff
@@ -0,0 +1,315 @@
+# RFC: Loose
+
+- Repo: [`babel/babel`](https://github.com/babel/babel)
+- Start Date: 2020-06-08
+- RFC PR: <!-- leave this empty, to be filled in later -->
+- Related Issues: <!-- if relevant -->
+- Authors: Nicolò Ribaudo
+- Champion: Nicolò Ribaudo
+- Implementors: <!-- the names of everyone who will work on the PR. you can leave this empty if you would like someone else to work on it -->
+
+# Summary
+
+Many of our plugins have a `loose` option, which enables the compiler to make some assumptions about the code you are writing. It uses these to ignore certain edge cases and generate a smaller or faster output.
+
+The `loose` option has been around since `6to5`: at first as a top level option, and then it was split into each plugin as a catch all for anything that we wanted to be less spec compliant.
+
+However, `loose` has some problems: namely that the term itself is not clear what it means, and _should_ be more related to a part of the language rather than to the plugin that it's compiling it (see the "Motivation" section for more details). A lot of this behavior is documented with a single example, so in the future we’ll want to be specific with the assumptions themselves and maybe provide a set of tests/codesandboxes explaining differences.
```

Comment:
```suggestion
However, `loose` has some problems: namely the term itself is not really descriptive at all and _should_ be named after the part of the language it affects rather than the plugin that's compiling it (see the "Motivation" section for more details). A lot of this behavior is documented with a single example, so in the future we’ll want to be specific with the assumptions themselves and maybe provide a set of tests/codesandboxes explaining the differences.
```